traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/edge/Node.java

```diff
@@ -89,9 +89,6 @@ public String getId() {
 
 	@SuppressWarnings({"PMD.CyclomaticComplexity", "PMD.NPathComplexity"})
 	public List<InetRecord> getIpAddresses(final JsonNode ttls, final Resolver resolver, final boolean ip6RoutingEnabled) {
```

Comment:
I think this is good.

A) we should never allow an external API to change `ipAddresses`.
B) I don't think we ever really have a good reason to use a `Resolver` (like `ZoneManager` here, which will default to using the system resolver if the fqdn is outside of TR's zone).

However, I now notice that `resolver` is unused in this method, so we should remove that parameter and fix all the places this method is called to no longer pass in `ZoneManager` as a resolver.